README.md

```diff
@@ -349,6 +350,40 @@ The available *options* are:
 
 If *validate* is not defined, [*text*.checkValidity](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#dom-cva-checkvalidity) is used. While the input is not considered valid, changes to the input will not be reported.
 
+### Textarea
+
+#### Inputs.textarea(*options*)
+
+[<img src="./img/textarea.png" alt="A Textarea asking for your biography" width="660">](https://observablehq.com/@observablehq/input-textarea)
+
+```js
+viewof bio = Inputs.textarea({label: "Biography", placeholder: "Tell us a little about yourself…"})
+```
+
+[Source](./src/textarea.js) · [Examples](https://observablehq.com/@observablehq/input-textarea) · A Textarea allows multi-line freeform text input. By default, a Textarea will report its value immediately on input. If more deliberate behavior is desired, the *submit* option can be set to true to wait until a button is clicked or Command-Enter is pressed.
```

Comment:
Command/Ctrl-Enter? I'm assuming ctrl-enter works on non-Mac?

Reply:
I didn’t change this documentation for this PR (I just moved it around), but yeah, it is assuming macOS here.